ghdeploy.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
Instead of creating this script, use the existing `publish_docs.sh` otherwise it would again lead to situations like #108 . ensure that the script is usable from both a webUI and travis. use idea similar to `generate.sh` to run conditional code based on whether running from a webUI.